.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Before creating an issue please make sure you are using the latest version of spotivy. -->
+
+# Bug report
+
+<!-- Issues which contain questions or support requests will be closed. -->
+
+**What is the current behavior?**
+
+
+**If the current behavior is a bug, please provide the steps to reproduce.**
+
+<!-- A great way to do this is to provide your configuration via a GitHub repository -->
+<!-- The most helpful is a minimal reproduction with instructions on how to reproduce -->
+<!-- Repositories with too many files are not suitable -->
+<!-- Please only add small code snippets directly into this issue -->
+<!-- https://gist.github.com is a good place for longer code snippets -->
+<!-- If your issue is caused by a dependency, please create an issue on the dependency repository instead -->
+
+**What is the expected behavior?**
+
+<!-- "It should work" is not a helpful explanation -->
+<!-- Explain exactly how it should behave -->
+
+**Other relevant information:**
+
+- Spotivy version:
+- Node.js version:
+- Operating System:
+- Additional tools:
```

Comment:
I can't think of anything that could be added as "additional tools". What did you think of? I believe we can remove it to keep things simple.